client.go

```diff
@@ -386,6 +390,26 @@ func (s *Client) LoopOut(globalCtx context.Context,
 	}, nil
 }
 
+// getExpiry returns an absolute expiry height based on the sweep confirmation
+// target, constrained by the server terms.
+func (s *Client) getExpiry(height int32, terms *LoopOutTerms,
+	confTarget int32) (int32, error) {
+
+	cltvDelta := confTarget
+
+	switch {
+	case confTarget < terms.MinCltvDelta:
+		cltvDelta = terms.MinCltvDelta
```

Comment:
could just return inline here then we don't need cltvDelta

Reply:
won't the linter complain?

Reply:
looks ok?
```
	switch {
	case confTarget < terms.MinCltvDelta:
		return height + terms.MinCltvDelta, nil

	case confTarget > terms.MaxCltvDelta:
		return 0, fmt.Errorf("...")
	}

	return height + confTarget, nil
```

Reply:
oh yes, true!

Reply:
fixed